### ISSUE TITLE
Django TextChoices causes a false-positive reportCallIssue error

### ISSUE THREAD

**Issue by hubert-springbok** [microsoft/pyright#7756]
**Describe the bug**
Since pyright version 1.1.351, we experience a false-positive error when defining a TextChoices enum. It seems pyright is expecting a function call, while in reality a tuple is assigned to a class attribute.

**Code or Screenshots**
```python
from django.db.models import TextChoices


class MyChoices(TextChoices):
    CHOICE = "CHOICE", "A choice"
```
django: 4.2.11
python: 3.10

Result: `/test_file.py:5:14 - error: Expected 1 positional argument (reportCallIssue)`

**VS Code extension or command-line**
Command line version 1.1.359 (occurs on all versions past 1.1.350 from my testing)
